AutocompleteClient/FW/Logic/Result/Recommendations/CIORecommendationsResponse.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  CIORecommendationsResponse.swift
+//  AutocompleteClient
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import Foundation
+
+public struct CIORecommendationsResponse{
+    public let pod: Pod
+    public let searchResponse: CIOSearchResponse
```

Comment:
Unfortunately, we likely need a different response format here.  Recommendations results have an additional property on the individual results `strategy`, and no groups or facets etc.